metrics/CSAF/DocumentCSAFRedRestricted/DocumentCSAFRedRestricted.yaml

```diff
@@ -0,0 +1,9 @@
+--- # Metadata
+- id: DocumentCSAFRedRestricted
+- description: This rule assesses whether a [SecurityAdvisoryDocument] that provides the property [RemoteDocumentLocation] is access protected by ensuring that the [noAuthentication] property is not utilized.
```

Comment:
There is a reference to p1 missing

Reply:
Okay, I would just find it more intuitive to reference [noAuthentication] with p1 and then set the p1 targetValue to false ("noAuthentication should not be set")